lamindb/dev/db/_add.py

```diff
@@ -38,22 +48,34 @@ def add(  # type: ignore  # no support of different naming of args across overlo
     Example:
 
     >>> # insert a new record
```

Comment:
How about, `# add a record (by passing a record)`

Reply:
ok!

Reply:
I think it'd be more readable to write it in two lines, making an experiment record, and passing it.

Reply:
ok! [](edited-with-Axolo)